src/ess/livedata/config/instruments/odin.py

```diff
@@ -63,7 +65,10 @@ def _make_odin_detectors() -> StreamLUT:
     Input keys based on
     https://confluence.ess.eu/display/ECDC/Kafka+Topics+Overview+for+Instruments
     """
-    return {InputStreamKey(topic='odin_detector', source_name='timepix3'): 'timepix3'}
+    # return {InputStreamKey(topic='odin_detector', source_name='timepix3'): 'timepix3'}
+    return {
+        InputStreamKey(topic='odin_detector_tpx3_empir', source_name='test'): 'test'
```

Comment:
```suggestion
        InputStreamKey(topic='odin_detector_tpx3_empir', source_name='test'): 'timepix3'
```
... at least this was the idea of this stream mapping: Isolate our software from upstream topic/source_name changes. Revert the changes higher up.

Reply:
I don't remember why, but yesterday I had to update the value too to get it to work? Or maybe I just messed it yesterday.